File: Liguagem C#/1006.cs
using System; 

class URI {

    static void Main(string[] args) { 
        
        double A = double.Parse(Console.ReadLine());
        double B = double.Parse(Console.ReadLine());
        double C = double.Parse(Console.ReadLine());
        
        double M = ((A * 2)+(B * 3)+(C * 5)) /10;
        
        Console.Write("MEDIA = {0:F1}\n",M);
        

    }

}

File: Linguagem Dart/1006.dart
import 'dart:io';

void main()
{
    double A = double.parse(stdin.readLineSync());
    double B = double.parse(stdin.readLineSync());
    double C = double.parse(stdin.readLineSync());
    
    double MEDIA = ((A * 2)+(B * 3)+(C * 5)) / 10;
    print("MEDIA = " + MEDIA.toStringAsFixed(1));
}
